### PR TITLE
New app: LIFX Lights

### PR DESCRIPTION
**Pull request by danielsitnik**
# Description
Show the status of your LIFX Lights!
This app connects to the LIFX API and displays information on your lights such as name, color, power state and brightness:
lifx_lights

Please check the `README.md` for more detailed information.

# Copilot

### Summary
📝💡🆕

This pull request adds a new app for the Tidbyt community called LIFX Lights, which allows users to control and display the colors of their LIFX smart bulbs on the Tidbyt device. It includes a `manifest.yaml` file for app registration and a `README.md` file for app documentation.

> _`LIFX Lights` app_
> _manifest and README files_
> _autumn leaves glow bright_

### Walkthrough
* Create and document a new app for controlling LIFX lights on the Tidbyt ([link](https://github.com/tidbyt/community/pull/1318/files?diff=unified&w=0#diff-7329e75d55cb2a9605b8725d315cba65677863ab6d83cdd768c1928474987eceR1-R8), [link](https://github.com/tidbyt/community/pull/1318/files?diff=unified&w=0#diff-13a0fb73955b668c0e915bf97ecee85a56203f0ae4d3de49b690af0169dd1bf9R1-R54))
  - Add a manifest file `manifest.yaml` with the app's metadata ([link](https://github.com/tidbyt/community/pull/1318/files?diff=unified&w=0#diff-7329e75d55cb2a9605b8725d315cba65677863ab6d83cdd768c1928474987eceR1-R8))
  - Add a README file `README.md` with the app's overview, requirements, configuration, error handling and demo mode ([link](https://github.com/tidbyt/community/pull/1318/files?diff=unified&w=0#diff-13a0fb73955b668c0e915bf97ecee85a56203f0ae4d3de49b690af0169dd1bf9R1-R54))
  - Include a GIF image of the app in action in the README file ([link](https://github.com/tidbyt/community/pull/1318/files?diff=unified&w=0#diff-13a0fb73955b668c0e915bf97ecee85a56203f0ae4d3de49b690af0169dd1bf9R1-R54))
  - Explain the code logic and the use of the `lifxlan` library in the README file ([link](https://github.com/tidbyt/community/pull/1318/files?diff=unified&w=0#diff-13a0fb73955b668c0e915bf97ecee85a56203f0ae4d3de49b690af0169dd1bf9R1-R54))

### CODE REVIEW

**Comment by danielsitnik** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by nolan-lynch** [comment]
Hey @danielsitnik

I have been building my own LIFX integration but with a different display. Mine doesn't show the current state (on/off, %) as yours. It only shows up in rotation if a bulb is offline and how long it has been since it was last seen.

Are you interested in merging the codes so that there is only 1 LIFX app in the store? The other option I was thinking of was calling mine "LIFX - Error Status Board" and keeping them separate. But honestly, I need to find out how Tidbyt devs are going to handle multiple integrations of the same type/company but with different views by different contributors. (@betterengineering do you have a recommendation?)

Thanks again. Sorry if this is a weird request. 

Nolan

File: apps/lifxlights/lifx_lights.star [hunk 1]
```diff
@@ -0,0 +1,437 @@
...
+            schema (schema): The configuration schema.
+    """
+
+    return schema.Schema(
+        version = "1",
+        fields = [
+            schema.Text(
+                id = "token",
+                name = "Personal Access Token",
+                desc = "Token generated at https://cloud.lifx.com.",
```
Comment (matslina) [inline apps/lifxlights/lifx_lights.star:120]: These descriptions are actually not visible to users in the iOS/Android app at the moment. Probably want to add the same information to the app description to help folks along.

**Comment by matslina** [comment]
On the topic of multiple apps of the same type or for the same company/product/API: the more the merrier! If we end up with 10 apps for LIFX then I would imagine one or two of them will have the perfect combinations of features that users want, and those are the ones that will primarily surface as users search or explore for apps to install.

**Pull request merged**